build-tests/s390/tumbleweed/test-image-MicroOS/appliance.kiwi

```diff
@@ -0,0 +1,278 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- OBS-Profiles: @BUILD_FLAVOR@ -->
+<image schemaversion="7.5" name="kiwi-test-image-MicroOS">
+    <description type="system">
+        <author>Marcus Schäfer</author>
+        <contact>marcus.schaefer@suse.com</contact>
+        <specification>MicroOS disk test build for IBM Secure Execution</specification>
+    </description>
+    <profiles>
+        <profile name="SUSE-Infra" description="MicroOS Confidental Compute Encrypted"/>
+        <profile name="IBM-Cloud-Secure-Execution-z16" description="MicroOS Confidental Compute Encrypted For IBM Cloud Regions Z16"/>
```

Comment:
```suggestion
        <profile name="IBM-Cloud-Secure-Execution-z16" description="MicroOS Confidental Compute Encrypted For IBM Cloud Regions z16"/>
```
It is a capital 'Z' except when followed by a version number.

Reply:
done